src/content/docs/specs/versioning/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: API versioning
+description: Details about how the API is versioned
+prev: false
+sidebar:
+   label: Versioning
+   order: 1
+---
+
+The Open Podcast API is subject to change over time. To ensure backwards compatibility and effective communication of server capabilities, the API specification MUST be versioned in a consistent way.
+
+The specification follows the [Semantic versioning model](https://semver.org/). Each version of the specification MUST adhere to the following rules:
+
+**Major** versions
+: A major version update indicates that a breaking change has occured. This is reserved for the deprecation, addition, or renaming of **core** capabilities.
+
+**Minor** versions
+: A minor version update indicates that a new **optional** feature has been added or that a **core** feature has received a non-breaking change. This can include the deprecation or addition of parameters or behaviors.
+
+**Patch** versions
+: A patch version update indicates that a small non-breaking change has been made to clarify a feature or address an issue with wording.
```

Comment:
Should we clarify here that Patch versions will not change the actual spec but only the documentation?